pushtx/broadcaster.go

```diff
@@ -68,6 +68,8 @@ type Broadcaster struct {
 
 	quit chan struct{}
 	wg   sync.WaitGroup
+
+	sync.Mutex
```

Comment:
Looks like we can avoid this mutex by instead making the copy of transactions before handing them over to the goroutine?

Reply:
Fixed! PTAL